README.md

```diff
@@ -26,30 +26,30 @@ Download the most recent version of [Sketch](https://www.sketchapp.com/).
 
 ## Getting Started
 
-**Welcome to the Summer ’18 Sketch Library document**
+**Welcome to the Summer ’18 Sketch Library kit**
 
-The Salesforce UX team has made quite a few changes to this document since the last release. We made optimizations so you can more efficiently create Salesforce Lightning interfaces. In the instructions below, we’ll cover the basics and some tips for you to get started with this Sketch Library document. You can still use this document just like you have in previous releases, but the Salesforce UX team believes this file is best used as a Sketch Library. What does that mean and why should you use it that way? We’ll answer those questions below.
+The Salesforce UX team has made quite a few changes to this kit since the last release. We made optimizations so you can more efficiently create Salesforce Lightning interfaces. In the instructions below, we’ll cover the basics and some tips for you to get started with this Sketch Library kit. You can still use this kit just like you have in previous releases, but the Salesforce UX team believes this file is best used as a Sketch Library. What does that mean and why should you use it that way? We’ll answer those questions below.
```

Comment:
Looks like theres more than 1 now, so - "believes these files are best used as Sketch Libraries"

Reply:
why should you use **them** that way

Reply:
Good catch. Fixed.